utils/index.js

```diff
@@ -192,6 +192,16 @@ const currencyFilter = (number, fractionDigits = 2) => {
   })
 }
 
+const sortByLineNumber = (...objToSort) => {
+  const superObj = Object.assign({}, ...objToSort);
+  const filteredObj = Object.entries(superObj).filter(obj => {
+    return typeof obj[1] === 'object' && obj[1] !== null && obj[1] !== undefined && obj[1].hasOwnProperty('line')
```

Comment:
Maybe it's just me, but I have trouble reading this. Should we add a comment that explains what we're doing (and why) so that in the future we know what it is?

![confused](https://user-images.githubusercontent.com/289193/61678086-e1528480-accf-11e9-94c1-ddbfd4ad0f1a.gif)

Also, maybe it's just me :)